pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[project]
+name = "huak"
+version = "0.0.4a0"
+description = "A Python package manager written in Rust inspired by Cargo."
+authors = [
+    {email = "cnpryer@gmail.com"},
+    {name = "Chris Pryer"}
+]
+readme = "README.md"
+license = {text = "MIT"}
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Rust",
+]
+
+[project.urls]
+Issues = "https://github.com/cnpryer/huak/issues"
+Documentation = "https://docs.rs/crate/huak"
+homepage = "https://github.com/cnpryer/huak"
+repository = "https://github.com/cnpryer/huak"
+
+[tool.maturin]
+bindings = "bin"
+
+[build-system]
+requires = ["maturin>=0.13,<0.14"]
+build-backend = "maturin"
```

Comment:
hmm so we did need a pyproject.toml. this is kinda what i envisioned in huak-py. im guessing the action needs it at the root of the project since it's not like we're creating a python package that binds to the rust implementation. regardless, more for posterity, i'd love to get this there next.

Reply:
It might be possible to put the pyproject.toml in huak-py, set the workflow directory to huak-py, and pass `-m ../Cargo.toml` arg to maturin actions command.

My thinking is that trying to troubleshoot that plus the added complexity for python binding huak doesn't have yet seemed like putting the cart before the horse.

Reply:
I'll just open an issue to document that we eventually want to do this at some point. I do agree with your reasoning. I'm a proponent for the process of get it done 👉 evaluate 👉 refine.